CPC-CHARTER.md

```diff
@@ -35,7 +35,7 @@ and direction of the policies as described in the CPC Policy document and
 approved by the Board.
 
 The Board sets the overall CPC policy through [Foundation mission and vision
-statements][] as well as within the [Foundation bylaws][]. The policy describes
+statements][https://openjsf.org/about/] as well as within the [Foundation bylaws][https://bylaws.openjsf.org]. The policy describes
```

Comment:
Rather than adding these inline here; if you update the x’s on lines 312-313, they’ll also work

Reply:
This has been updated per your requested changes.